frontend/www/js/omegaup/omegaup.ts

```diff
@@ -79,6 +79,11 @@ export namespace omegaup {
     String = 'string',
   }
 
+  export enum CountdownFormat {
+    EVENT_COUNTDOWN,
+    WAIT_BETWEEN_UPLOADS_SECONDS,
```

Comment:
oh! perdón, no me di cuenta que aquí se espera que los valores de los enums vayan en CamelCase u_u se puede cambiar a

```suggestion
    EventCountdown,
    WaitBetweenUploadsSeconds,
```

Reply:
Listo